lib/libzpool/Makefile.am

```diff
@@ -27,6 +27,9 @@ AM_CFLAGS += $(ZLIB_CFLAGS)
 
 AM_CFLAGS += -DLIB_ZPOOL_BUILD
 
+# libtool flag
```

Comment:
I'd suggest adding a comment that this is not really meant for a compiler but `libtool` and it circumvents autotools deficiencies. It is a gigantic hack but still better than people wasting time on troubleshooting failed compilations.
Should we report this to Mr. Stallman?

Reply:
Sure.

I suspect Mr. Stallman would respond with rude words for daring to not use the GPL on something.

Reply:
I just wrote following to bug-automake@gnu.org

---
Dear automake developers!

A standard `libtool` invocation line generated by automake looks like:
```
LTCOMPILE = $(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
        $(LIBTOOLFLAGS) --mode=compile $(CC) $(DEFS) \
        $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) \
        $(AM_CFLAGS) $(CFLAGS)
```
Sometimes files compiled using the method above make the compiler emit errors. Those errors are suppressed by default which makes troubleshooting impossible. `libtool` has a command line option, `-no-suppress` which can be used to make the compiler verbose. Unfortunately, there is no way to inject that option since `libtool` demands that it comes after `--mode=compile`. `AM_LIBTOOLFLAGS` nor `LIBTOOLFLAGS` cannot be used for that purpose since "it is too early", according to `libtool`'s command line parser. It is somewhat possible to use `AM_CFLAGS` for that purpose but then it breaks modes other than `--mode=compile`.

Is there any way to make libtool more verbose?
Thanks in advance!

Software used:
```
autoconf (GNU Autoconf) 2.71
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<https://gnu.org/licenses/gpl.html>, <https://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.
automake (GNU automake) 1.16.5
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv2+: GNU GPL version 2 or later <https://gnu.org/licenses/gpl-2.0.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Tom Tromey <[tromey@redhat.com](mailto:tromey@redhat.com)>
       and Alexandre Duret-Lutz <[adl@gnu.org](mailto:adl@gnu.org)>.
```